plugin/FileInfo.h

```diff
@@ -35,7 +35,7 @@ class FileInfo : public QObject
     Q_PROPERTY(QDateTime modifiedDate READ modifiedDate NOTIFY sourceChanged)
 
 public:
-    explicit FileInfo(QObject* parent = 0);
+    explicit FileInfo(QObject *parent = 0);
```

Comment:
I think it's a bit out of scope as non UI element. Additionally haven't seen similar implementations, so might not be that needed.